### ISSUE TITLE
Crazy feature: read aloud microsub channel

### ISSUE THREAD

**Issue by swentel** [swentel/indigenous-android#193]

**Comment by EdwardHinkle** [comment]
I have that as a todo in iOS version as well! Haha great minds!

**Comment by EdwardHinkle** [comment]
I also have the idea of activating it via Siri on iOS. Maybe you can register with the Google Assistant as well?

**Comment by swentel** [comment]
Yes I think so. I added this because opened a request at a some point for caching of microsub because he was on the tube, but now switched to taking his bike. So if he could listen to his feed while biking, that would be a great replacement :)

Have to look into it, but should be relatively easy indeed. I'm not sure though what I'm going to pass to the assistant, as in, which properties of a feed item: author, title and content I guess are the most relevant ones ?
